### DISCUSSION TITLE
Xrip with Canada freestyle libre 2 sensors

### DISCUSSION THREAD

**Discussion by jaysondennis** [Navid200/xDrip#84] [Libre]
Hi All!

Does Xrip now work with Canada freestyle libre 2 sensors?  I haven't heard specifically about Canada version recently and use Diabox app but much prefer Xdrip and do not use nightscout.... please let me know.

Thanks in advance!!!

**Comment by Navid200** [comment]
@tzachi-dar Would you please advise?

**Reply by jaysondennis** [reply]
@tzachi-dar any thoughts?

Thanks!!

**Comment by tzachi-dar** [comment]
Unfortunately it is not suported by xdrip.

בתאריך יום ו׳, 29 באוק׳ 2021, 00:36, מאת jaysondennis ‏<
***@***.***>:

**Reply by Navid200** [reply]
Thanks

**Reply by jaysondennis** [reply]
Thats to bad.... the european version of the sensors do work with xdrip in my understanding so i thought work would be underway to make the U.S. and canada ones usable...

Any future plans or other suggestions besides Diabox?

Thanks!!
